src/components/interfaces/MOBILE_API.xml

```diff
@@ -3102,6 +3164,252 @@
+        </element>
+        <element name="EXTERIOR_REAR_LIGHTS" value="902">
+            <description>Include exterior lights located at the back of the vehicle. For example, license plate lights, reverse lights, cargo lights, bed lights an trailer assist lights.</description>
+        </element>
+        <element name="EXTERIOR_LEFT_LIGHTS" value="903">
+            <description>Include exterior lights located at the left side of the vehicle. For example, left puddle lights and spot lights.</description>
+        </element>
+        <element name="EXTERIOR_RIGHT_LIGHTS" value="904">
+            <description>Include exterior lights located at the right side of the vehicle. For example, right puddle lights and spot lights.</description>
+        </element>
+    </enum>
+
+    <enum name="LightStatus">
+        <element name="ON"/>
+        <element name="OFF"/>
+    </enum>
+
+    <struct name="LightCapabilities">
+        <param name="name" type="LightName" mandatory="true" />
+        <!-- Assuming light ON/OFF status is always available -->
+        <param name="densityAvailable" type="Boolean" mandatory="false">
+            <description>
+            Indicates if the light's density can be set remotely (similar to a dimmer).
+            </description>
+        </param>
+        <param name="RGBColorSpaceAvailable" type="Boolean" mandatory="false">
+            <description>
+            Indicates if the light's color can be set remotely by using the RGB color space.
+            </description>
+        </param>
+    </struct>
+
+    <struct name="LightControlCapabilities">
+        <param name="moduleName" type="String" maxlength="100" mandatory="true">
+            <description>
+            The short friendly name of the light control module.
+            It should not be used to identify a module by mobile application.
+            </description>
+        </param>
+        <param name="supportedLights" type="LightCapabilities" minsize="1" maxsize="100" array="true" mandatory="true">
+        <description> An array of available light names that are controllable. </description>
+        </param>
+    </struct>
+
+    <struct name="LightState">
+        <param name="id" type="LightName" mandatory="true">
+            <description>The name of a light or a group of lights. </description>
+        </param>
+        <param name="status" type="LightStatus" mandatory="true"/>
+        <param name="density" type="Float" minvalue="0" maxvalue="1" mandatory="false" />
+        <param name="color" type="RGBColor" mandatory="false" />
+    </struct>
+
+    <struct name="LightControlData">
+        <param name="lightState" type="LightState" mandatory="true" minsize="1" maxsize="100" array="true">
+            <description>An array of LightNames and their current or desired status. No change to the status of the LightNames that are not listed in the array.</description>
+        </param>
+    </struct>
+
+    <enum name="DisplayMode">
```

Comment:
Add `since="5.0"` to new enums

Reply:
@jacobkeeler Fixed at commit fd9f349304e4d47d25d1fed79a0442a599717de8